### PR TITLE
Atlas Tokens Readme file

### PR DESCRIPTION
**Pull request by wibjorn**
task-377248
preview-93

No changeset needed, updating documentation only.

### CODE REVIEW

File: css/tokens/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Atlas CSS Tokens
+
+A design tokens is a definition of a design-related variable, such as one that affects color, typography, or spacing. Design tokens must be shareable and fairly platform agnostic. Atlas's design tokens are written in Scss (buzzword: source of truth!), and though a [build process](.\tokens\index.js), translated and published into JSON so they can be more easily shared by solutions that do not use Scss.
+
+## Access our design tokens
+
+Firstly, in order to access these tokens, you must have installed atlas-css in your project. [See installation steps]('../../../README.md). Once this is complete, you'll need to decide whether you want to reference get them in Scss or JSON.
```
Comment (jdanyow) [inline css/tokens/README.md:7]: to reference the

File: css/tokens/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Atlas CSS Tokens
+
+A design tokens is a definition of a design-related variable, such as one that affects color, typography, or spacing. Design tokens must be shareable and fairly platform agnostic. Atlas's design tokens are written in Scss (buzzword: source of truth!), and though a [build process](.\tokens\index.js), translated and published into JSON so they can be more easily shared by solutions that do not use Scss.
```
Comment (jdanyow) [inline css/tokens/README.md:3]: `./tokens/index.js` for portability

File: css/tokens/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+@import './node_modules/@microsoft/atlas-css/src/tokens/index.scss';
+// OR if using a bundler like Parcel you may just reference the package directly
+@import '@microsoft/atlas-css/src/tokens/index.scss';
+```
+
+### Get JSON from NPM
+
+In NodeJs, referencing our `node_modules` will also allow you access [JSON](http://unpkg.com/browse/@microsoft/atlas-css/dist/tokens.json), which lives at `/dist/tokens.json`.
+
+You can also access the [TypeScript typings](https://unpkg.com/browse/@microsoft/atlas-css/dist/tokens.d.ts) in the same directory. Note the JSDoc comment in the example below.
```
Comment (jdanyow) [inline css/tokens/README.md:24]: this is nice.

**Review by jdanyow — CHANGES_REQUESTED** [review]

**Review by ollips — APPROVED** [review]

File: css/tokens/README.md (hunk 1 above)
Comment (wibjorn) [reply css/tokens/README.md]: updated

File: css/tokens/README.md (hunk 2 above)
Comment (wibjorn) [reply css/tokens/README.md]: updated

**Review by jdanyow — APPROVED** [review]

**Pull request merged**